scripts/toggles.py

```diff
@@ -4,11 +4,50 @@
 import collections
 import re
 import logging
+from enum import Enum
 
 
 LOGGER = logging.getLogger(__name__)
 logging.basicConfig(level=logging.INFO)
 
+
+class ToggleTypes(Enum):
+    WAFFLE_FLAG = "WaffleFlag"
+    WAFFLE_SWITCH = "WaffleSwitch"
+    WAFFLE_SAMPLE = "WaffleSample"
+    EXPERIMET_WAFFLE_FLAG = "ExperimentWaffleFlag"
+    COURSE_WAFFLE_FLAG = "CourseWaffleFlag"
+    COURSE_WAFFLE_FLAG_COURSE_OVERRIDE = "WaffleUtilsWaffleflagcourseoverridemodel"
```

Comment:
This is still confusing to me, because we are mixing data here and there is only one toggle type for CourseWaffleFlag.

I think we should have:
```
# This could also be named ModelTypes
class ToggleModelTypes(Enum):
    WAFFLE_FLAG = "waffle.flag"
    WAFFLE_SWITCH = "waffle.switch"
    WAFFLE_SAMPLE = "waffle.sample"
    COURSE_WAFFLE_FLAG_COURSE_OVERRIDE = "WaffleUtilsWaffleflagcourseoverridemodel"

# This could be a dict map (simplest), or it could be a function.
MODEL_TO_TOGGLE_TYPE_MAP = {
    ToggleModelTypes.WAFFLE_FLAG: ToggleTypes.WAFFLE_FLAG,
    ...
    ToggleModelTypes.COURSE_WAFFLE_FLAG_COURSE_OVERRIDE: ToggleTypes.WAFFLE_FLAG,
}
```

Reply:
Hmm, yes, there is definitely a confusion in terminology here. I will get back to you in a sec about this